api/v1alpha/api.pb.go

```diff
@@ -224,7 +224,7 @@ type Image struct {
 	// Base format of the image, required. This indicates the original format
 	// for the image as nowadays all the image formats will be transformed to
 	// ACI.
-	BaseFormat *ImageFormat `protobuf:"bytes,1,opt,name=base_format" json:"base_format,omitempty"`
+	BaseFormat *ImageFormat `protobuf:"bytes,1,opt,name=base_format,json=baseFormat" json:"base_format,omitempty"`
```

Comment:
That's what upstream https://github.com/golang/protobuf geneates, I had no intuition to what version the generated code was pinned at.

Since travis failed I have one now, namely the version referenced in Godeps, pinning the script to the very same version now and see if the generated code is the same.